.travis.yml

```diff
@@ -6,6 +6,7 @@ otp_release:
 sudo: true
 before_script:
   - sudo pip install crossbar
+script: "mix test && mix spell.example.pubsub"
```

Comment:
I see, I thought that `after_script` was before `after_success`on the lifecycle and it was returning an error when failing... I believe it's good for now. If we add more examples we probably need to trigger them in a different way without modifying `.travis.yml` file all the time